test/rekt/features/kafka_sink.go

```diff
@@ -84,6 +76,13 @@ func BrokerWithTriggersAndKafkaSink(env environment.Environment) *feature.Featur
 	eventTrigger2.SetType(trigger2FilterType)
 
 	f := feature.NewFeatureNamed("Trigger with KafkaSink")
+	topic, err := apisconfig.DefaultFeaturesConfig().ExecuteBrokersTopicTemplate(metav1.ObjectMeta{Name: brokerName, Namespace: env.Namespace()})
+
+	f.Assert("No error creating broker topic", func(ctx context.Context, t feature.T) {
+		if err != nil {
+			t.Fatal(err)
+		}
+	})
```

Comment:
I'm not entirely sure about this new test here, what's the test goal here?

Reply:
I guess it doesn't necessarily make sense as a test, I was just trying to add some kind of check to the error which may be returned by `ExecuteBrokersTopicTemplate`. I'll switch it to just panic if there is an error, because there really shouldn't be an error here.